packages/fx-core/resource/package.nls.json

```diff
@@ -590,10 +603,10 @@
   "core.aadAppQuestion.description": "Your Azure Active Directory app for Single Sign On",
   "core.teamsAppQuestion.label": "Teams app",
   "core.teamsAppQuestion.description": "Your Teams app", 
-  "core.M365SsoLaunchPageOptionItem.label": "SSO enabled personal tab",
-  "core.M365SsoLaunchPageOptionItem.detail": "Identity aware webpages embedded in Microsoft Teams, Outlook and the Microsoft 365 app",
-  "core.M365SearchAppOptionItem.label": "Search-based message extension",
-  "core.M365SearchAppOptionItem.detail": "Insert search result when user composes messages in Microsoft Teams or emails in Outlook",
+  "core.M365SsoLaunchPageOptionItem.label": "React with Fluent UI",
+  "core.M365SsoLaunchPageOptionItem.detail": "A web app that uses Fluent UI React components to get a Teams look and feel",
+  "core.M365SearchAppOptionItem.label": "Collect Form Input and Process Data",
```

Comment:
Other labels only capitalize the first character, need to be consistent. This label is in CamelCase,

Reply:
@sffamily Thanks, I missed this one and fixed now.